kof-operator/internal/controller/clusterdeployment_kof_cluster_role.go

```diff
@@ -25,6 +28,8 @@ const ClusterDeploymentGenerationKey = "cluster_deployment_generation"
 const RegionalClusterNameKey = "regional_cluster_name"
 const RegionalDomainKey = "regional_domain"
 
+const KOF_ISTIO_SECRET_TEMPLATE = "kof-istio-secret-template"
```

Comment:
I've found recently a valid reason why Go does not use UPPER_CASE for [constants](https://go.dev/ref/spec#Constant_expressions) and [prefers](https://go.dev/doc/effective_go#mixed-caps) mixedCaps/MixedCaps instead: because a constant may also be either [Exported](https://go.dev/ref/spec#Exported_identifiers) or notExported, depending on its first letter case. In this case it may be used in other modules so let's make it explicitly Exported:

```suggestion
const KofIstioSecretTemplate = "kof-istio-secret-template"
```